src/game_interpreter.cpp

```diff
@@ -4643,6 +4645,29 @@ bool Game_Interpreter::CommandManiacCallCommand(lcf::rpg::EventCommand const&) {
 	return true;
 }
 
+bool Game_Interpreter::CommandSetFlag(lcf::rpg::EventCommand const& com) {
+
+	std::string flagName = Utils::LowerCase(ToString(com.string));
+	int flagValue = ValueOrVariable(com.parameters[0], com.parameters[1]);
+
+	if (flagName == "rpg2k3commands") //2k3 commands on 2k games
```

Comment:
Please remove the ``//`` comments. It is obvious what these conditions do.

Reply:
comments are gone